client/my-sites/email/email-providers-comparison/price/google-workspace.tsx

```diff
@@ -87,9 +90,14 @@ const GoogleWorkspacePrice = ( {
 				</div>
 			) }
 
-			{ isEligibleForFreeTrial && (
+			{ isEligibleForFreeTrial && ( introductoryOffer?.cost_per_interval ?? 0 ) === 0 && (
 				<div className="google-workspace-price__trial-badge badge badge--info-green">
-					{ translate( '1 month free' ) }
+					{ translate( '%(intervalCount)d %(intervalUnit)s free', {
+						args: {
+							intervalCount: introductoryOffer?.interval_count,
+							intervalUnit: introductoryOffer?.interval_unit,
+						},
+					} ) }
```

Comment:
Since we now support different intervals, we would need to [take that into account](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md#pluralization) when localizing this string.